python/src/worker/tasks/differential_expression.py

```diff
@@ -41,7 +41,7 @@ def _format_request(self):
             "baseCells": [int(x) for x in baseCells],
             "backgroundCells": [int(x) for x in backgroundCells],
             "genesOnly": self.task_def.get("genesOnly", False),
-            "comparisonType": self.task_def.get("comparisonType", "within")
+            "comparisonType": self.task_def.get("comparisonType", "within"),
```

Comment:
```suggestion
            "comparisonType": self.task_def.get("comparisonType", "within")
```
```suggestion
            "comparisonType": self.task_def.get("comparisonType", "within"),
```

Reply:
The trailing comma is added by linter

Reply:
You can activate the linter for the python worker by opening the python workspace (i.e. `code python.code-workspace`)